src/OrbitGl/TimerTrack.cpp

```diff
@@ -175,7 +177,7 @@ void TimerTrack::DrawTimer(TextBox* prev_text_box, TextBox* current_text_box,
                         function_id == highlighted_function_id;
 
   static const Color kHighlightColor(100, 181, 246, 255);
```

Comment:
Please erase this constant if you are not using it :)

Reply:
Done. Thanks! I didn't notice this when rebasing after PR #1844.